docs/tutorials/autogluon-cloud.md

```diff
@@ -1,2 +1,119 @@
+    instance_type="INSTANCE_TYPE"  # Checkout supported instance and pricing here: https://aws.amazon.com/sagemaker/pricing/
+)
+```
+
+To retrieve general info about the fitted `CloudPredictor`
+
+```{.python}
+cloud_predictor.info()
+```
+
+### Reattach to a Previous Training Job
+If your local connection to the training job died for some reason, i.e. lost internet connection, your training job will still be going on SageMaker, and you can reattach to the job with another `CloudPredictor` as long as you have the job name.
+
+```{.python}
+another_cloud_predictor = TabularCloudPredictor(cloud_output_path='YOUR_S3_BUCKET_PATH')
+another_cloud_predictor.attach_job(job_name="JOB_NAME")
+```
+
+To be noticed, the reattached job will no longer give live stream of the training job's log. Instead, the log will be available once the training job is finished.
+
+## Endpoint Deployment and Real-time Prediction
+If you want to deploy a predictor as a SageMaker endpoint, which can be used to do real-time inference later, it is just one line of code:
+
+```{.python}
+cloud_predictor.deploy(
+    instance_type="INSTANCE_TYPE"  # Checkout supported instance and pricing here: https://aws.amazon.com/sagemaker/pricing/
+)
+```
+
+You can also attach to a deployed endpoint,
+
+```{.python}
+cloud_predictor.attach_endpoint(endpoint="ENDPOINT_NAME")
+```
+
+To do real-time prediction,
+
+```{.python}
+result = cloud_predictor.predict_real_time(
+    'test.csv',
+    instance_type="INSTANCE_TYPE"  # Checkout supported instance and pricing here: https://aws.amazon.com/sagemaker/pricing/
+)  # result will be a pandas dataframe
+```
+
+## Batch Inference
+When minimizing latency isn't a concern, then the batch inference functionality may be easier, more scalable, and more appropriate.
+To do a batch inference,
+
+```{.python}
+cloud_predictor.predict('test.csv')  # results will be stored in s3 bucket
+cloud_predictor.download_predict_results()  # download the results to your local machine
+```
+
+## Convert the CloudPredictor to a Local AutoGluon Predictor
+You can easily convert the `CloudPredictor` you trained on SageMaker to your local machine as long as you have the same version of AutoGluon installed locally.
+
+```{.python}
+local_predictor = cloud_predictor.to_local_predictor()  # local_predictor would be a TabularPredictor
+```
+
+## Note on Image Modality
+If your training and inference tasks involve image modality, you need to
+1. Make sure your column contains **absolute paths** to the images
+2. provide argument `image_column` as the column name containing image paths to `CloudPredictor` fit/inference APIs.
+
+```{.python}
+cloud_predictor.fit(..., image_column="IMAGE_COLUMN_NAME")
+cloud_predictor.predict_real_time(..., image_column="IMAGE_COLUMN_NAME")
+cloud_predictor.predict(..., image_column="IMAGE_COLUMN_NAME")
+```
+
+## What Containers AutoGluon Cloud Support?
+`autogluon.cloud` supports containers starting AutoGluon DLC 0.6.0.
```

Comment:
What is an AutoGluon DLC?
What does DLC mean?

Reply:
I'll expand it